candle-binding/src/classifiers/unified.rs

```diff
@@ -1024,6 +1025,45 @@ impl DualPathUnifiedClassifier {
             model_type
         };
 
+        // Validate model availability and fall back if necessary
```

Comment:
maynot be the best option to fallback to different model. Would you mind disabling Gemma test and model download in CI?